ibis/backends/dask/tests/test_client.py

```diff
@@ -0,0 +1,112 @@
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pandas.util.testing as pandas_tm
+import pytest
+from dask.dataframe.utils import tm
+from pytest import param
+
+import ibis
+
+from .. import connect
+from ..client import DaskTable  # noqa: E402
+
+pytestmark = pytest.mark.dask
+
+
+def make_dask_data_frame():
+    return dd.from_pandas(pandas_tm.makeDataFrame(), npartitions=1)
+
+
+@pytest.fixture
+def client():
+    return connect(
+        {
+            'df': dd.from_pandas(
+                pd.DataFrame({'a': [1, 2, 3], 'b': list('abc')}),
+                npartitions=1,
+            ),
+            'df_unknown': dd.from_pandas(
+                pd.DataFrame({'array_of_strings': [['a', 'b'], [], ['c']]}),
+                npartitions=1,
+            ),
+        }
+    )
+
+
+@pytest.fixture
+def table(client):
+    return client.table('df')
+
+
+def test_client_table(table):
+    assert isinstance(table.op(), ibis.expr.operations.DatabaseTable)
+    assert isinstance(table.op(), DaskTable)
+
+
+def test_client_table_repr(table):
+    assert 'DaskTable' in repr(table)
+
+
+def test_load_data(client):
+    client.load_data('testing', make_dask_data_frame())
+    assert client.exists_table('testing')
+    assert client.get_schema('testing')
+
+
+@pytest.mark.xfail(reason="Need to fix up create_table")
```

Comment:
can you add the TODO issue (can be same one for anything you are xfailing / marking TODO)

Reply:
Do you prefer new issues, or can I continue to update https://github.com/ibis-project/ibis/issues/2537 with more detail?

Reply:
i think have a tracking issue is good here (with the check boxes)

Reply:
https://github.com/ibis-project/ibis/issues/2553